hart/utils.py

```diff
@@ -28,18 +25,6 @@ def get_cloud_init_template(template_name='minion.sh'):
     return environment.get_template(template_name)
 
 
-def get_saltstack_repo_url(debian_codename, salt_branch):
```

Comment:
I think we need to implement the new way of pinning salt branches if we can no longer do it through the repo url by adding an apt pin like in the example:
```
Package: salt-*
Pin: version 3007.*
Pin-Priority: 1001
```
in `/etc/apt/preferences.d/salt-pin-1001`.

Reply:
Updated